src/konnektor/tests/test_toy_data.py

```diff
@@ -131,12 +131,12 @@ def test_build_n_random_mst_network(n_sub_networks, overlap):
                 assert len(net.nodes) == n_compounds // n_sub_networks + overlap
 
 
-def build_random_fully_connected_network():
+def test_build_random_fully_connected_network():
     n_compounds = 30
     mst_network = build_random_fully_connected_network(
```

Comment:
the name `mst_network` is confusing here, since it's not an MST - maybe a follow-up issue (not sure how prevealent this problem is)

Reply:
agreed, done.